vulnfeeds/test_data/nvdcve-2.0/CVE-2014-1424.json

```diff
@@ -0,0 +1 @@
+{"resultsPerPage":1,"startIndex":0,"totalResults":1,"format":"NVD_CVE","version":"2.0","timestamp":"2025-09-08T00:43:09.297","vulnerabilities":[{"cve":{"id":"CVE-2014-1424","sourceIdentifier":"security@ubuntu.com","published":"2014-11-24T15:59:00.090","lastModified":"2025-04-12T10:46:40.837","vulnStatus":"Deferred","cveTags":[],"descriptions":[{"lang":"en","value":"apparmor_parser in the apparmor package before 2.8.95~2430-0ubuntu5.1 in Ubuntu 14.04 allows attackers to bypass AppArmor policies via unspecified vectors, related to a \"miscompilation flaw.\""},{"lang":"es","value":"apparmor_parser en el paquete apparmor anterior a 2.8.95~2430-0ubuntu5.1 en Ubuntu 14.04 permite a atacantes evadir las políticas AppArmor a través de vectores no especificados, relacionado con un 'fallo en miscompilación.'"}],"metrics":{"cvssMetricV2":[{"source":"nvd@nist.gov","type":"Primary","cvssData":{"version":"2.0","vectorString":"AV:N\/AC:L\/Au:N\/C:N\/I:P\/A:P","baseScore":6.4,"accessVector":"NETWORK","accessComplexity":"LOW","authentication":"NONE","confidentialityImpact":"NONE","integrityImpact":"PARTIAL","availabilityImpact":"PARTIAL"},"baseSeverity":"MEDIUM","exploitabilityScore":10.0,"impactScore":4.9,"acInsufInfo":false,"obtainAllPrivilege":false,"obtainUserPrivilege":false,"obtainOtherPrivilege":false,"userInteractionRequired":false}]},"weaknesses":[{"source":"nvd@nist.gov","type":"Primary","description":[{"lang":"en","value":"CWE-264"}]}],"configurations":[{"nodes":[{"operator":"OR","negate":false,"cpeMatch":[{"vulnerable":true,"criteria":"cpe:2.3:a:ubuntu:apparmor:*:*:*:*:*:*:*:*","versionEndIncluding":"2.8.94-0ubuntu1.4","matchCriteriaId":"A060E745-4598-4736-8901-45387CAFB265"},{"vulnerable":true,"criteria":"cpe:2.3:o:canonical:ubuntu:14.04:*:*:*:lts:*:*:*","matchCriteriaId":"F5CFE100-7387-4823-BBED-9E932DC808B4"}]}]}],"references":[{"url":"http:\/\/www.ubuntu.com\/usn\/USN-2413-1","source":"security@ubuntu.com","tags":["Patch","Vendor Advisory"]},{"url":"http:\/\/www.ubuntu.com\/usn\/USN-2413-1","source":"af854a3a-2127-422b-91ae-364da2661108","tags":["Patch","Vendor Advisory"]}]}}]}
```

Comment:
Probably format this to make it more readable. It gets loaded into JSON anyway so formatting should not matter.